alerts/hbase/slow-operations.json

```diff
@@ -0,0 +1,35 @@
+{
+    "displayName": "HBase - Slow Operations",
+    "documentation": {
+        "content": "Slow operations can be an indication that the HDFS may be in an unhealthy state. ",
```

Comment:
Can we share what is the threshold that needs to be breached for the slow operation alert to trigger?